restock.lic

```diff
@@ -1,58 +1,65 @@
 #   Documentation: https://elanthipedia.play.net/Lich_script_repository#restock
-
 custom_require.call(%w[common common-items common-money])
-
 class Restock
   include DRC
   include DRCM
   include DRCI
-
   def initialize
     setup
-
     items = parse_restockable_items
+
+    custom_loc_items = items.select{ |k| k['hometown'] }
+
+    items -= custom_loc_items
+    restock_items(items)
+
+    unique_hometowns = custom_loc_items.map{ |k| "#{k['hometown']}" }.uniq
```

Comment:
❓ Why is there string interpolation here instead of mapping to `k['hometown']` directly?

Reply:
Not sure, but I fixed it.